webapp/src/app.tsx

```diff
@@ -133,6 +133,13 @@ export class ProjectView
     private openingTypeScript: boolean;
     private preserveUndoStack: boolean;
 
+    private highContrastSubscriber: data.DataSubscriber = {
```

Comment:
>DataSubscriber [](start = 41, length = 14)

This component already extends data.Component. Do you need this here too?

Reply:
Yes, because we need to get updates outside of the render lifecycle. The data.Component only updates the component by calling render